clients/oathkeeper/dart/pubspec.yaml

```
@@ -1,7 +1,8 @@
 name: ory_oathkeeper_client
 version: 0.40.2
-description: OpenAPI API client
+description: "OpenAPI API client for Ory's cloud native Identity & Access Proxy / API (IAP) and Access Control Decision API that authenticates, authorizes, and mutates incoming HTTP(s) requests."
```

```suggestion
description: "OpenAPI API client for Ory Oathkeeper, Ory's cloud native Identity & Access Proxy / API (IAP) and Access Control Decision API that authenticates, authorizes, and mutates incoming HTTP(s) requests."
```

---

This might now be too long 🤔

---

```suggestion
description: "OpenAPI API client for Ory Oathkeeper, a cloud native Identity & Access Proxy and Access Control Decision API that authenticates, authorizes, and mutates incoming HTTP(s) requests."
```

This should be just right, 180 characters 😊